src/app/modules/login/login-routing.module.ts

```diff
@@ -1,10 +1,16 @@
 import { NgModule } from '@angular/core';
-import { RouterModule, Routes } from '@angular/router';
+import { Routes } from '@angular/router';
+import { LoginComponent } from './components/login/login.component';
+import { LoginModule } from './login.module';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'login', component: LoginComponent },
+];
 
 @NgModule({
-  imports: [RouterModule.forChild(routes)],
-  exports: [RouterModule]
+  declarations: [LoginComponent],
+  imports: [],
+  exports: [LoginModule, LoginComponent],
```

Comment:
In exports you define components, directives, pipes, modules which possibly can be reused in other modules. Is there is any chance that LoginComponent can be used somewhere outside LoginModule?)

And in general, I don's see any reason to have another login-routing.module file. As i can see, you've already declare LoginComp in LoginModule, which is right, provide needed component for path 'login' in app-routing.module. All is left to do is import LoginModule in app.module